onos-e2t/Chart.yaml

```diff
@@ -7,7 +7,7 @@ name: onos-e2t
 description: ONOS E2 Termination
 kubeVersion: ">=1.17.0"
 type: application
-version: 1.3.2
+version: 1.3.3
 appVersion: v0.10.0
```

Comment:
app version? I think it should be 0.10.3

Reply:
Done